docs/index.md

```diff
@@ -1,9 +1,10 @@
-<div>
-  <div class="center logo">
-    ![logo](manual/asset/logo-small.png)
```

Comment:
Why the change from div to p? Wouldn't replacing `manual/asset/logo-small.png` with `docs/images/logo-small.png` be enough?

But even with the other changes, I would recommend making it an internal link so it's not tied to the main branch. If you get what I mean

Reply:
There's no specific reason i changed to p, but it's my common approach to center something in markdown

Reply:
Changed to internal link